### PR TITLE
Feature/mobileapp docs

### PR DESCRIPTION
**Pull request by compojoom**

### CODE REVIEW

File: mobileapp/localizations.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: Localizations
+---
+
+:::note
+This doc is for maintainers or for people who want to setup a similar translation process with a copy of the trustlines app.
+:::
+
+
+The Trustlines mobile app uses the localization framework [FBT](https://facebookincubator.github.io/fbt/) and the translation platform [Crowdin](https://crowdin.com/) for supporting different languages. The utility scripts in `mobileapp/translation/scripts` are used for enabling a smooth integration of the mentioned tools. The scripts `crowdinToFbt.js`, `fbtToCrowdin.js` and `splitTranslation.js` are copied from the [Origin Protocol mobile app repository](https://github.com/OriginProtocol/origin/tree/master/mobile) and modified to our needs. The respective license can be found in `<root>/translation/LICENSE`
```
Comment (dakingha69) [inline mobileapp/localizations.md:10]:
```suggestion
The Trustlines App uses the localization framework [FBT](https://facebookincubator.github.io/fbt/) and the translation platform [Crowdin](https://crowdin.com/) for supporting different languages. The utility scripts in `/translation/scripts` are used for enabling smooth integration of the mentioned tools. The scripts `crowdinToFbt.js`, `fbtToCrowdin.js`, and `splitTranslation.js` are copied from the [Origin Protocol mobile app repository](https://github.com/OriginProtocol/origin/tree/master/mobile) and modified to our needs. The respective license can be found in `/translation/LICENSE`
```

File: mobileapp/testing/e2e.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+---
```
Comment (dakingha69) [inline mobileapp/testing/e2e.md:1]: Duplicate of `mobileapp/e2e_testing.md`

**Review by dakingha69 — COMMENTED** [review]
Docs looking nice 👍 I added some suggestions in my review comments. Mostly typo fixes and wording.

File: mobileapp/customizations/custom_icons.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+1. Go to https://icomoon.io/app.
+2. Click on the button **Import Icons**.
+3. Select the `ico-moon-config.json` under `<PROJECT_ROOT>/src/shared/components/common/icons/ico-moon-config.json`.
+
+## How to update the icon font
+
+1. Follow the previous steps on how to browse available icons.
+2. Add, edit or remove icons and generate the new icon font.
+3. Put the generated `icomoon.ttf` into `<PROJECT_ROOT>/assets/fonts`.
+4. Rename the generated `selection.json` to `ico-moon-config.json` and put the file into `<PROJECT_ROOT>/src/shared/components/common/icons`.
```
Comment (compojoom) [inline mobileapp/customizations/custom_icons.md:25]:
```suggestion
4. Rename the generated `selection.json` to `ico-moon-config.json` and put the file into `/src/shared/components/common/icons`.
```

File: mobileapp/known_issues/android.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+How to completely integrate a native module might vary and usually is described in the modules repository description. Always recheck / diff your files for changes after you call `react-native link <packageName>` and if it reflects what you anticipated.
+
+## Android Simulator
+When running the application in the android simulator, you can not easily use the camera and interact with the app as you would do with a physical phone. You can send remote commands to the simulator though.
+To open the QR code link to create a new trustline, you can start the VIEW intent, passing the URL copied from the QR code screen.
+
+```adb shell am start -a "android.intent.action.VIEW" -d "<QR Code URL here>"```
+
+This basically does the same as clicking the link inside the messages app.
```
[the remark this answers is not in the document]
Comment (compojoom) [reply mobileapp/known_issues/android.md:85]: Well, yes. Okay, that is an option, but not an easier option than the `adb shell` thingy :)

**Review by dakingha69 — APPROVED** [review]

**Pull request merged**